app/common/ipc-protocol.ts

```diff
@@ -68,29 +68,37 @@ export const Err = t.intersection([
   })
 ])
 
+export enum ErrCodes {
+  ParseError = -32700,
+  InvalidRequest = -32600,
+  MethodNotFound = -32601,
+  InvalidParams = -32602,
+  InternalError = -32603
+}
+
```

Comment:
Thanks for taking them out into its own Enum. It was my only objection :sweat_smile:

Reply:
I can read your mind xD